inc/database.class.php

```diff
@@ -0,0 +1,320 @@
+         "date_lastboot", [
+            'value'      => $this->fields['date_lastboot'],
+            'maybeempty' => true
+         ]);
+      echo "</td></tr>\n";
+
+      echo "<tr class='tab_bg_1'>";
+      echo "<td><label for='is_active$rand'>".__('Is active')."</label></td>";
+      echo "<td>";
+      Dropdown::showYesNo('is_active', $this->fields['is_active']);
+      echo "<td>" . __('Associable to a ticket') . "</td><td>";
+      Dropdown::showYesNo('is_helpdesk_visible', $this->fields['is_helpdesk_visible']);
+      echo "</td></tr>\n";
+
+      echo "<tr class='tab_bg_1'>";
+      echo "<td><label for='dropdown_groups_id_tech$rand'>".__('Group in charge of the hardware')."</label></td>";
+      echo "<td>";
+      Group::dropdown([
+         'name'      => 'groups_id_tech',
+         'value'     => $this->fields['groups_id_tech'],
+         'entity'    => $this->fields['entities_id'],
+         'condition' => ['is_assign' => 1],
+         'rand' => $rand
+      ]);
+
+      echo "</td>";
+
+      $rowspan        = 3;
+
+      echo "<td rowspan='$rowspan'><label for='comment'>".__('Comments')."</label></td>";
+      echo "<td rowspan='$rowspan' class='middle'>";
+
+      echo "<textarea cols='45' rows='".($rowspan+2)."' id='comment' name='comment' >".
+         $this->fields["comment"];
+      echo "</textarea></td></tr>";
+
+      echo "<tr class='tab_bg_1'>";
+      echo "<td><label for='dropdown_users_id_tech$rand'>".__('Technician in charge of the hardware')."</label></td>";
+      echo "<td>";
+      User::dropdown(['name'   => 'users_id_tech',
+         'value'  => $this->fields["users_id_tech"],
+         'right'  => 'own_ticket',
+         'entity' => $this->fields["entities_id"],
+         'rand'   => $rand]);
+      echo "</td></tr>";
+      echo "<tr><td><label for='dropdown_manufacturers_id$rand'>".Manufacturer::getTypeName(1)."</label></td>";
+      echo "<td>";
+      Manufacturer::dropdown(['value' => $this->fields["manufacturers_id"], 'rand' => $rand]);
+      echo "</td></tr>\n";
+
+      if ($this->isNewItem($ID)) {
+         echo "<tr class='tab_bg_1'>";
+         echo "<td colspan='4' class='center'>".__('Default instance')."</td>";
+         echo "</tr>";
+         echo "<tr class='tab_bg_1'>";
+         echo "<td><label for='instance_port$rand'>"._n('Port', 'Ports', 1)."</label></td>";
+         echo "<td>";
+         echo Html::input(
+            'instance_port', [
+               'type' => 'number',
+               'id' => 'instance_port'.$rand
+            ]
+         );
+
+         echo "</td>";
+         echo "<td><label for='dropdown_is_onbackup$rand'>".__('Has backup')."</label></td>";
+         echo "<td>";
+         Dropdown::showYesNo('is_onbackup', 0, -1, ['rand' => $rand]);
+         echo "</td></tr>\n";
+      }
+
+      $this->showInventoryInfo();
+
+      $this->showFormButtons($options);
+
+      return true;
+   }
+
+   function rawSearchOptions() {
+
+      $tab = parent::rawSearchOptions();
+
```

Comment:
Some more search options could be usefull.

Reply:
Sure, sounds like I forget that :)